docs/suspensive.org/src/pages/en/docs/react-query/QueryClientConsumer.mdx

```diff
@@ -0,0 +1,128 @@
+import { Callout } from '@/components'
+
+# QueryClientConsumer
+
+<Callout type='experimental'>
+
+`<QueryClientConsumer/>` is an experimental feature, meaning this interface is subject to change.
+
+</Callout>
+
+This component allows the use of `useQueryClient` within JSX while bypassing React hook rules. It facilitates declarative access to the QueryClient in legacy class components or specific environments (such as SSR and micro-frontends).
+
+The props interface varies depending on the installed version of `@tanstack/react-query`. If version 4 is installed, you will use `props.context`, and for version 5, `props.queryClient` is used.
+
```

Comment:
The queryClient prop is unlikely to be used very often, so I think it's appropriate to add an example that doesn't use any props as the first example.

Reply:
I agree. However, including general usage examples might make the documentation a bit lengthy. Would it be okay to simplify the examples for the props?

```tsx
```

Reply:
I reply in https://github.com/toss/suspensive/pull/1315#discussion_r1802225377